micromamba/src/repoquery.cpp

```diff
@@ -110,10 +116,17 @@ set_common_search(CLI::App* subcom, bool is_repoquery)
                 format = QueryResultFormat::kPRETTY;
             }
 
-            // if (ctx.json)
-            //     format = QueryResultFormat::kJSON;
+            if (config.at("json").compute().value<bool>())
+            {
+                format = QueryResultFormat::kJSON;
+            }
 
-            repoquery(qtype, format, local, specs[0]);
+            auto& channels = config.at("channels").compute().value<std::vector<std::string>>();
+            if (!channels.empty())
+            {
+                use_local = false;
+            }
```

Comment:
This can be simpified in `use_local = use_local && channels.empty()` or even `use_local = channels.empty()`